src/plan_expand_hypertable.c

```diff
@@ -51,6 +51,7 @@ typedef struct CollectQualCtx
 	List *restrictions;
 	FuncExpr *chunk_exclusion_func;
 	List *join_conditions;
```

Comment:
would be nice to add comments explaining the differences between these qual lists